### PR TITLE
Feature/app view

### PR DESCRIPTION
**Pull request by hughwin**
What up homies! 

Basically, in this PR I've created the AppView and the AppMenu classes off of the UML diagram. 

Main points: 

- We now have a clickable menu bar at the top of the AppView. 
- Clicking on the buttons will (once the functionality has been added) take you to different views of different parts of the application). I would have implemented these, but I wasn't sure whose responsibility it was. 

image

### CODE REVIEW

**Comment by hughwin** [comment]
Ah, forgot to mention that I gold plated it slightly, in so far as there are mneumonincs. Essentially, you can get to the main menu by pressing alt+ enter, and select menu items with "t" for teacher etc.

File: src/main/java/se/appview/AppMenu.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+import se.controller.MenuActionListener;
+
+import javax.swing.*;
+import java.awt.event.KeyEvent;
+
+public class AppMenu extends JMenuBar {
+
+    private static final String MENU_NAME = "Main menu";
+    public static final String FIRST_MENU_ITEM = "Teachers";
```
Comment (gsears) [inline src/main/java/se/appview/AppMenu.java]: I'd rename FIRST_MENU_ITEM as it's not particularly semantic outside of this class, as in, when I read it in the switch statement in the menu action listener class I was like 'erm...what does that do again?'

Maybe something like: 
LOAD_TEACHER_PAGE
LOAD_COURSES_PAGE
would be better?

File: src/main/java/se/appview/AppMenu.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    public static final String SECOND_MENU_ITEM = "Courses";
+    private MenuActionListener mal = new MenuActionListener();
+
+
+    public AppMenu() {
+
+        JMenu menu = new JMenu(MENU_NAME);
+        menu.setMnemonic(KeyEvent.VK_M); // Uses a to open the menu. Press Alt + M.
+
+        JMenuItem firstMenuItem = new JMenuItem(FIRST_MENU_ITEM, KeyEvent.VK_T);
```
Comment (gsears) [inline src/main/java/se/appview/AppMenu.java]: same for variable names, firstMenuItem isn't as readable as 'miLoadTeacherView' or something like that.

**Comment by gsears** [comment]
Looks good, just a few requests with regard to readability in the files!

**Pull request merged**